.github/workflows/ci.yml

```diff
@@ -6,6 +6,12 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
+      - name: Setup ImageMagick
```

Comment:
Is this change required for the unit tests?

Reply:
Correct. for the context load Integration test. If not the build fails. It will need to setup this as well in the jenkins build.